README.md

```diff
@@ -100,6 +102,29 @@ $dateFormatter->format(new DateTime(), 'eeee dd LLLL Y - HH:mm:ss');
 // example output: zaterdag 02 juni 2040 - 05:57:02
 ```
 
+It is also possible to format dates and times to relative dates, such as 'today' and 'tomorrow'
+The RelativeDateFormatOptions decides how many days ahead it will try to convert a date to a relative date.
+
+```php
+$dateFormatOptions = new DateFormatOptions('nl_NL', date_default_timezone_get())
+$dateFormatter = new DateFormatService($dateFormatOptions);
+
+$dateFormatter->formatRelative(time(), 'Y-m-d', new RelativeDateFormatOptions(1));
+// example output: Vandaag
+
+$dateFormatter->formatRelative(new DateTime('+1 day'), 'Y-m-d', new RelativeDateFormatOptions(1);
+// example output: Morgen
```

Comment:
Is `Gisteren` also possible with this library? If so, add that as additional example